acceptance/framework/consul/cli_cluster.go

```diff
@@ -264,7 +264,7 @@ func (c *CLICluster) SetupConsulClient(t *testing.T, secure bool) *api.Client {
 	consulClient, err := api.NewClient(config)
 	require.NoError(t, err)
 
-	return consulClient
+	return consulClient, config.Address
```

Comment:
Now also returning the address so that we can dial it in the test to get the cert and check the expiration.